docs/docs/cmd/spo/list/list-roleinheritance-break.md

```diff
@@ -11,13 +11,16 @@ m365 spo list roleinheritance break [options]
 ## Options
 
 `-u, --webUrl <webUrl>`
-: URL of the site where the list to retrieve is located
+: URL of the site.
```

Comment:
Is there a reason why we shortened the original description? The original one was pretty descriptive